core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.ts

```diff
@@ -102,6 +128,21 @@ export class DotEditPageStateControllerSeoComponent implements OnChanges {
         }
     }
 
+    ngOnInit(): void {
+        this.dotPropertiesService
+            .getKey(this.featureFlagEditURLContentMap)
+            .pipe(take(1))
+            .subscribe((result) => {
+                this.featureFlagEditURLContentMapIsOn = result && result === 'true';
+
+                if (this.featureFlagEditURLContentMapIsOn && this.pageState.params.urlContentMap) {
+                    this.menuItems = this.getMenuItems();
+                }
+
+                this.options = this.getStateModeOptions(this.pageState);
+            });
```

Comment:
This code is duplicated. Since the `featureFlagEditURLContentMapIsOn` is a feature flag that only affects the `edit page` portlet. Do we want to make part of the `DotPageStateService`?

Reply:
We will get rid of the duplicated toolbar when SEO is done. I discussed that with @fmontes and is not worth the effort